src/main/resources/localization/messages-tr.properties

```diff
@@ -0,0 +1,80 @@
+error.permissions.user = Bu komutu kullanabilmek için sunucuyu yönet izininin olması gerek!
+error.permissions.perms_or_role = Bu komudu kullanabilmek için sunucuyu yönet izinin ya da {0} rolünün olması gerek!
+error.permissions.perms_no_role = Bu komudu kullanabilmek için sunucuyu yönet izninin olması gerek! Bir çekiliş manajeri kurulmamıştır. Bir admin bunu `{0}` ile yapabilir
+error.permissions.bot = Bu kanalda birkaç eksik yetkim var: `Mesajları Oku`, `Mesaj Gönder`, `Mesaj Geçmişini Oku`, `Bağlantı Yerleştir`. Bu izinleri bu kanalda düzeltin, ya da botu [buradan]({0}) yeniden davet edin.
+error.invalid_time.format = Verdiğin `{0}` değerini geçerli bir süreye çeviremedim!
+error.invalid_time.min = Verdiğin ({0}) süresi minimum olan ({1}) süresinden kısaydı!
+error.invalid_time.max = Verdiğin ({0}) süresi maximum olan ({1}) süresinden uzundu!
+error.max_giveaways.guild = Zaten bu sunucuda {0} çekiliş devam ediyor!
+error.max_giveaways.channel = Zaten bu kanalda {0} çekiliş devam ediyor!
+error.invalid_winners.format = Verdiğin `{0}` değerini geçerli bir kazanan sayısına çeviremedim!
+error.invalid_winners.max = Verdiğin ({0}) kazanan sayısı 1 ile {1} arasında değildi!
+error.invalid_prize.length = Ödül uzunluğu {0} karakterden daha uzun olamaz!
+error.invalid_descr.length = Açıklama uzunluğu {0} karakterden daha uzun olamaz!
+error.invalid_emoji.length = Özel düğme yazısı {0} karakterden uzun olamaz!

```

Comment:
error.invalid_emoji.length = Buton içeriği {0} karakterden uzun olamaz!

Reply:
Doesn't matter buton is more englishy düğme is more Turkish either way correct might change tho